src/olympia/devhub/templates/devhub/new-landing/components/overview.html

```diff
@@ -1,11 +1,13 @@
-<section class="DevHub-Header">
-  <h1>{{ _('Customize Your Firefox') }}</h1>
-  <p>
-    {% trans %}
-    Add-ons let millions of Firefox users enhance their browsing experience.
-    If you know <a href="#">HTML</a>, <a href="#">JavaScript</a>, and <a href="#">CSS</a>,
-    you already have all the necessary skills to make a great add-on.
-    {% endtrans %}
-  </p>
-  <a href="#" class="Button Button--primary">{{ _('Learn How to make an Add-on') }}</a>
+<section class="DevHub-Overview content scheme-medium">
+  <div>
```

Comment:
I'll revisit this, it made various things easier but afair I removed the specific needs for the nested `<div>` on the plane ride to some degree so it should work with merging this with `<section>` or just make all `<section>`'s a `<div>` and use that for a more generic component-type.